docs/get-started/lotus/chain.md

```diff
@@ -108,7 +108,11 @@ lotus daemon --import-snapshot --halt-after-import <filename>
 
 ## Compacting the chain data
 
-It is possible to "prune" the current chain data used by Lotus to reduce the disk footprint of a Lotus node by resyncing from a minimal snapshot. This is a quick overview of the steps:
+It is possible to "prune" the current chain data used by Lotus to reduce the disk footprint of a Lotus node by resyncing from a minimal snapshot.
+
+> Note: A minimal snapshot only has the full state trees from some recent tipset onward and nothing before that tipset. To have all historical states, sync from the full snapshot above.
```

Comment:
Do not apply: the above is incorrect, both snapshots have just the last 2000 states: if you want really everything-everything - the snapshot .car would be ~350GiB and it still doesn't quite include everything: https://github.com/filecoin-project/lotus/issues/3849

The difference between pruned/unpruned is how many of the *headers* are present. You can validate the large one via `--import-chain`, you can not do this with the smaller one.

Reply:
oh damn.. thats new to me and unfortunate...